insights/core/__init__.py

```diff
@@ -241,6 +241,37 @@ def get(self, item, default=None):
         return self.data.get(item, default)
 
 
+class CommandParser(Parser):
+    """
+    This class checks output from the command defined in th spec to be sure line
+    do not contain certain string values and that there is only one line.
```

Comment:
@lhuett you may need to check spelling, and is it checking that there is only one line or that if there is one line, it doesn't contain the bad strings?  Be sure to check the output documentation to see if it looks like you expected.